### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by AntonellaMorittu**
#### Description

* Add partials (navbar and footer) to web pages
* Add Sass stylesheet
* Create Home Button 

#### What I have learned working on this feature: 
* Style the page with Scss and HAML
* Add partials to a webpage

schermata 2018-03-22 alle 16 18 49

### CODE REVIEW

**Comment by holgertidemand** [comment]
Looks good!

**Review by holgertidemand — APPROVED** [review]
Looks good!

File: app/views/layouts/application.html.haml [hunk 1]
```diff
@@ -2,11 +2,13 @@
 %html
   %head
     %meta{content: "text/html; charset=UTF-8", http: {equiv: "Content-Type"}}
+    %link{:href => "https://fonts.googleapis.com/css?family=Limelight", :rel => "stylesheet"}
```
Comment (tochman) [inline app/views/layouts/application.html.haml:5]: Please do not use old hash syntax (I will not comment on this anymore, just silently reject PR's that contain this code)

File: features/create_articles.feature [hunk 2]
```diff
@@ -4,11 +4,15 @@ Feature: Create articles
   I would like to be able to create articles
 
   Background:
-    Given I visit the Homepage
+    Given the following user exists
+      | email           | password    | password_confirmation |
+      | harald@norge.no | OsloOslo123 | OsloOslo123           |
+    And I am logged in as "harald@norge.no"
```
Comment (tochman) [inline features/create_articles.feature:10]: Now that you have Active Admin, I assume that the article creation process will be moved to that interface, right?

**Review by tochman — CHANGES_REQUESTED** [review]

**Pull request closed without merging**